### PR TITLE
[MWPW-154335] [callout] Spacing issue encountered when the call-out section is added

### PR DESCRIPTION
**Pull request by rohitsahu**
- Adjust heigh of body if a card has callout text
- Callout text will now be inside one single div. It has been done to keep the height calculation simpler and for consistency with other blocks
- renamed 'callout-text' slot to 'callout-content'
- css fixes
- UTs fixes for above changes

Resolves: [MWPW-154335](https://jira.corp.adobe.com/browse/MWPW-154335)
https://jira.corp.adobe.com/browse/MWPW-155176 

**Test URLs:**
Before: https://main--milo--adobecom.hlx.live/drafts/rosahu/rosahu?martech=off
After: https://mwpw-154335--milo--rohitsahu.hlx.live/drafts/rosahu/rosahu?martech=off

### CODE REVIEW

File: libs/blocks/merch-card/merch-card.js [hunk 1]
```diff
@@ -249,6 +249,12 @@ const parseContent = async (el, merchCard) => {
     if (mnemonicList) bodySlot.append(mnemonicList);
   });
 
+  if (calloutContainer.children.length > 0) {
+    const calloutSlot = createTag('div', { slot: 'callout-content' });
```
Comment (Axelcureno) [inline libs/blocks/merch-card/merch-card.js:253]: Why not flatten the DOM and just add slot attribute to `calloutContainer`? 
```suggestion
    callloutContainer.setAttribute('slot', 'callout-content');
```

Comment (Axelcureno) [reply libs/blocks/merch-card/merch-card.js:253]: If it's like that already in mas code, let's leave it as is.

**Review by Axelcureno — APPROVED** [review]
Small nit comment but not a blocker.

**Comment by Axelcureno** [comment]
Please resolve comments @rohitsahu

**Review by rahulgupta999 — APPROVED** [review]

**Review by npeltier — APPROVED** [review]

**Comment by rohitsahu** [comment]
Milo PR has been validated by Priyanka, she does not have access, hence I have added "verified" label

**Comment by rahulgupta999** [comment]
Added Ready for stage label on behalf of @adgaurgit (PTO) and Priyanka

**Pull request merged**